agb/src/lib.rs

```diff
@@ -13,6 +13,9 @@
 #![feature(alloc_error_handler)]
 #![feature(allocator_api)]
 #![feature(asm_const)]
+#![feature(const_trait_impl)]
+#![allow(incomplete_features)] // generic_const_exprs, used for bg mode traits
+#![feature(generic_const_exprs)]
```

Comment:
hmmm.... We've been trying to reduce the number of unstable features we use as much as possible... How necessary is the fancy feature you've used?